src/components/Molecules/Task/index.jsx

```diff
@@ -60,13 +63,17 @@ const StyledNameAndButtonWrapper = styled.div`
   align-items: center;
   gap: 10px;
   width: 100%;
+  min-width: 0;
 `;
 
 const StyledTaskName = styled.div`
   font-family: ${FONT_FAMILY.NOTO_SANS};
   ${TEXTS.S}
   color: ${COLOR.LIGHT_GRAY};
-  width: 100%;
+  flex: 1 1 auto;
+  min-width: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
```suggestion
  flex: 1 1 auto;
  min-width: 0;
  overflow: hidden;
  text-overflow: ellipsis;
  text-wrap: no-wrap;
```
text-wrapの指定を追加すると、3点リーダーで省略されるようになります！

参考：https://on-ze.com/archives/267

![image](https://github.com/MoriguchiHinata/Hinata-frontend-basic-2023-todoapp/assets/83479175/895f3d83-7454-4010-8e92-c13920db3226)